lib/cypress-rails/server.rb

```diff
@@ -78,6 +80,7 @@ def boot
           raise "Rack application timed out during boot" if timer.expired?
 
           @server_thread.join(0.1)
+          @initializer_hooks.run(:after_server_start)
```

Comment:
I placed the hook to call here as it appeared to be both the soonest, and arguably most safe place to ensure that the application server has booted.

Reply:
seems good